modules/social_features/social_profile/social_profile.module

```diff
@@ -376,7 +376,15 @@ function social_profile_preprocess_profile(array &$variables) {
     $variables['profile_image_url'] = ImageStyle::load('social_medium')
       ->buildUrl($profile->field_profile_image->entity->getFileUri());
   }
-
+  // Profile information URL.
+  $variables['profile_info_url'] = Url::fromRoute('view.user_information.user_information', ['user' => $profile->getOwnerId()]);
+
+  // Prepare variables for statistic block.
+  /** @var \Drupal\social_profile\UserStatistics $user_statistics */
+  $user_statistics = \Drupal::service('social_profile.user_statistics');
+  $variables['profile_topics'] = $user_statistics->nodeCount($user->id(), 'topic');
+  $variables['profile_events'] = $user_statistics->nodeCount($user->id(), 'event');
+  $variables['profile_groups'] = count(\Drupal::service('social_group.helper_service')->getAllGroupsForUser($user->id()));
```

Comment:
❗️  This now runs for all view modes and can have a negative effect on performance on user listings (where this data is not used).

Reply:
This was fixed in PR #1680